server/mediaserver.go

```diff
@@ -59,11 +59,12 @@ var BroadcastJobVideoProfiles = []ffmpeg.VideoProfile{ffmpeg.P240p30fps4x3, ffmp
 var AuthWebhookURL string
 
 type rtmpConnection struct {
```

Comment:
Do we still need the following fields within `rtmpConnection`? I think we can remove these and the associated code:

* needOrch
* eof
* sess
* lock

We can remove startSession/startSessionListener as well. All this has been subsumed by the session manager.

Note that the startSessionListener has an important property that the session manager doesn't currently have: it only allows for one refresh in-flight at a given time.

Reply:
Really great observations! Helping me be more vigilant about looking at existing code and reviewing which has become superfluous